db/migrate/20190803025721_modify_user_to_add_data_to_uid_and_provider.rb

```diff
@@ -0,0 +1,11 @@
+class ModifyUserToAddDataToUidAndProvider < ActiveRecord::Migration[5.2]
+  def change
+    users = User.all
+    provider = "twitter"
+    users.each do |user|
```

Comment:
It's generally better to use - `User.find_each`  instead of `User.all` followed by `each` - `for_each` saves allocating too many objects at once by not taking everything form db at once.